Sources/Network/URLSessionNetworkStack.swift

```diff
@@ -187,8 +189,7 @@ public extension Network {
                     return strongSelf.perform(request: authenticatedRequest, resource: resource, completion: completion)
 
                 case let .failure(error):
-                    completion(.failure(.authenticator(error.error)))
-
+                    completion(.failure(Network.Error(type: .authenticator(error.error), response: nil)))
```

Comment:
we should send the `response` anyway

Reply:
At this point the request hasn't been performed, right?

Reply:
Oops, you're right!
In that case, the `.authenticator` error shouldn't even have a `response` 😉